viz-lib/src/components/visualizations/editor/ContextHelp.tsx

```diff
@@ -53,5 +53,18 @@ function DateTimeFormatSpecs() {
   );
 }
 
+function TickFormatSpecs() {
+  const { HelpTriggerComponent } = visualizationsSettings;
+  return (
+    <HelpTriggerComponent
+      title="Tick Formatting"
+      href="https://redash.io/help/user-guide/visualizations/formatting-axis-ticks"
```

Comment:
In the [corresponding update to the website](https://github.com/getredash/website/pull/680), the slug is `formatting-axis`. I'll update this change to match